### PR TITLE
Wip small fix

### PR DESCRIPTION
**Pull request by LancelotMarti**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,9 +74,38 @@ Let's continue our journey by installing Gramine:
...
+- Intel DCAP library: This binary library provides functionality for dcap quote generation and verification. With ubuntu 22.04, it is required to install different packages to setup DCAP attestation. See [here](https://docs.oasis.io/node/run-your-node/prerequisites/set-up-trusted-execution-environment-tee/#:~:text=AESM%3A%20error%2030%E2%80%8B&text=Ensure%20you%20have%20all%20required,Attestation%20or%20EPID%20attestation%20sections.) for more details (section DCAP attestation).
+
+```console
+sudo apt update
+sudo apt install sgx-aesm-service libsgx-aesm-ecdsa-plugin libsgx-aesm-quote-ex-plugin libsgx-dcap-default-qpl
+```
+
+- Intel QPL library: This binary library provides functionality for the communication with DCAP verification services. One need to change the Intel QPL configuration file at this location `/etc/sgx_default_qcnl.conf`. Azure gives an example of a configuration file [here](https://learn.microsoft.com/en-us/azure/security/fundamentals/trusted-hardware-identity-management). This configuration can be use for testing in Azure:
```
Comment (damienbfs) [inline README.md]: One needS to change

Comment (damienbfs) [inline README.md]: Can be useD for testing

Comment (LancelotMarti) [reply README.md]: ah merci! oups

**Review by damienbfs — APPROVED** [review]
Thanks a lot!

**Pull request merged**